frontend/src/pages/devices/List.tsx

```diff
@@ -68,7 +72,7 @@ const List = () => {
       return <Loading />
     }
     return (
-      <table className="table table-responsive-sm">
+      <Table responsive>
```

Comment:
Did you mean to kill off the `sm`?

Reply:
Yes, it breaks it on mobile devices.
How it is with sm:
![image](https://user-images.githubusercontent.com/54916525/199119582-bdf6128c-6e09-439a-8b37-9a3f0579ddd9.png)

How it is without:
![image](https://user-images.githubusercontent.com/54916525/199119788-fe9d7cfd-dc81-4c36-b854-dfaacd62fa14.png)

Reply:
Should we do it for the users table too?

Reply:
Yeah, and I did. None of the main pages tables break on mobile

Reply:
I don't think you have:
<img width="378" alt="image" src="https://user-images.githubusercontent.com/12345527/200094231-d4054709-df9a-45c5-acf4-b3939d1b54e2.png">

Reply:
I have it on my local branch, should have specified.

Reply:
https://github.com/anthonywittig/mLock/pull/37/commits/8ce150ae4e97075209bb9b090d5ca201a5e48314